ert3_examples/snake_oil/run_demo

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This script serves both as an example, and for CI purposes
+
+set -e
+
+ert3 clean --all  # Only relevant when run manually
+
+# Let ERT draw and run a stochastic ensemble:
+ert3 run evaluation
+ert3 export evaluation
+# Assert that we have FOPT in the output
+grep FOPT experiments/evaluation/data.json >/dev/null
+
+
+# DOE (design of experiment)
+design2params 0 resources/designmatrix.xlsx designsheet defaults  # [semeio]
+python designmatrixtxt2json.py  # [hack]
+ert3 record load BPR_138_PERSISTENCE resources/designed_bpr_138_persistence.json
+ert3 record load BPR_555_PERSISTENCE resources/designed_bpr_555_persistence.json
+ert3 record load OP1_DIVERGENCE_SCALE resources/designed_op1_divergence_scale.json
+ert3 record load OP1_OCTAVES resources/designed_op1_octaves.json
+ert3 record load OP1_OFFSET resources/designed_op1_offset.json
+ert3 record load OP1_PERSISTENCE resources/designed_op1_persistence.json
+ert3 record load OP2_DIVERGENCE_SCALE resources/designed_op2_divergence_scale.json
+ert3 record load OP2_OCTAVES resources/designed_op2_octaves.json
+ert3 record load OP2_OFFSET resources/designed_op2_offset.json
+ert3 record load OP2_PERSISTENCE resources/designed_op2_persistence.json
+ert3 run doe
+ert3 export doe
+grep FOPT experiments/doe/data.json >/dev/null
```

Comment:
I would then include test for FOPR too...

Reply:
Fixed.